### PR TITLE
add styling

### PR DESCRIPTION
**Pull request by 29td**
In  this project am adding style to my leaderboard to make it look beautiful.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -2,6 +2,10 @@
   font-family: "Josefin Sans", sans-serif;
 }
 
+body {
+  background-color: aquamarine;
+}
+
 h1 {
```
Comment (DammyShittu) [inline src/style.css:9]: - [x] Your app styling is expected to follow the wireframe. The inputs overlap with the scores list. In the wireframe, there is some space between the scores listing and the scores inputs - as if they belong in different halves of the page. Kindly implement that.

  Your styling | Wireframe
  :---:|:---:
  ld-2 | ld-3

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @29td,

## Highlights

✔️ Good styling

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] You included `npm start` as a command in your package.json file, but you are missing the appropriate npm package for that. Kindly install it as well.

  ld-1

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (29td) [reply src/style.css:9]: _I think for the npm start the installation process is required in your computer, below is my npm start_

Screenshot 2022-07-27 183318

_The add scores is not overlaping in a 100% zoom_

Screenshot (33)

**Review by mukhammadrizooff — APPROVED** [review]
Hi @29td, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Project working fine ✔️ 
- Used project requirements well ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**